Examples/Example-iOS/AppDelegate.swift

```diff
@@ -55,7 +53,7 @@ final class AppDelegate: UIResponder, UIApplicationDelegate {
             }
         }
         let attributes = ["My Attribute":"My Attribute Value"]
-        BacktraceClient.shared?.addBreadcrumb("My Native Breadcrumb",
+        _ = BacktraceClient.shared?.addBreadcrumb("My Native Breadcrumb",
```

Comment:
does it make sense to assign a method result to nothing?

Reply:
This is how to ignore a return value in Swift:
https://developer.apple.com/forums/thread/71943